packages/wonder-blocks-modal/components/modal-content.js

```diff
@@ -8,22 +8,40 @@ import ModalHeader from "./modal-header.js";
 
 type Props = {
     header?: React.Element<typeof ModalHeader> | React.Node,
+    scrollOverflow: boolean,
     children: React.Node,
     style?: any,
```

Comment:
comments

Reply:
I'm not doing comments/docs until we can get this landed and then I'll do them all at once.